public/app/features/dashboard-scene/panel-edit/VizPanelManager.tsx

```diff
@@ -103,6 +102,8 @@ export class VizPanelManager extends SceneObjectBase<VizPanelManagerState> {
       $variables: variables,
       panel: sourcePanel.clone(),
       sourcePanel: sourcePanel.getRef(),
+      pluginId: sourcePanel.state.pluginId,
+      skipDataQuery: config.panels[sourcePanel.state.pluginId].skipDataQuery,
```

Comment:
We need these properties in state to avoid relying on `getPlugin` around panelEdit.